meta-intel-edison-distro/recipes-core/initrdscripts/initramfs-framework/init

```diff
@@ -121,6 +121,21 @@ boot_root() {
         return 1
     fi
 
+    # copy fstab
+    if [ -h /realroot/etc/fstab ] || [ -f /realroot/etc/fstab ]; then
```

Comment:
Better w/o bashisms, i.e. `if [ -h ... -a -f ... ]; then`.

Reply:
I'll consider.

Reply:
I don't know what to do here. I read:

>  Binary -a and -o are inherently ambiguous. Use 'test EXPR1 && test EXPR2' or 'test EXPR1 || test EXPR2' instead.

and

> Note the shell logical primitives also benefit from short circuit
operation, which can be significant for file attribute tests.

How about changing to:
`if [[ -h /realroot/etc/fstab || -f /realroot/etc/fstab ]]; then`

I think this should work for bash, ash and ksh. Correct?